x-pack/docs/en/security/troubleshooting.asciidoc

```diff
@@ -319,6 +320,65 @@ In this case, you must install the
 <<ciphers, JCE Unlimited Strength Jurisdiction Policy Files>>.
 --
 
+[[trb-security-kerberos]]
+=== Common Kerberos exceptions
+
+*Symptoms:*
+
+* User authentication fails due to either GSS negotiation failure
+or a service login failure on the server side or in the {es} HTTP client. Some of
+the common exceptions are listed below with some tips to resolve them. 
+
+*Resolution:*
+
+`Failure unspecified at GSS-API level (Mechanism level: Checksum failed)`::
```

Comment:
Are these error messages in specific logs or on specific machines in the deployment?